vertx-mongo-client/src/main/java/io/vertx/ext/mongo/package-info.java

```diff
@@ -351,7 +351,10 @@
  *
  *
  * `db_name`:: Name of the database in the mongoDB instance to use. Defaults to `default_db`
- * `useObjectId`:: Toggle this option to support persisting and retrieving ObjectId's as strings. Defaults to `false`.
+ * `useObjectId`:: Toggle this option to support persisting and retrieving ObjectId's as strings. If `true', hex-strings will
+ * be saved as native Mongodb ObjectId types in the document collection. This will allow the sorting of documents based on creation
+ * time. You can also derive the creation time from the hex-string using ObjectId::getDate(). Set to `false' for other types of your choosing.
```

Comment:
I would add that by default false will generate hex strings if the user does not provide an _id